run_coverage.py

```
@@ -0,0 +1,5 @@
+import subprocess, glob
+
+test_files = glob.glob('test/**/*_test.dart', recursive=True)
+cmd = ['flutter', 'test', '--coverage'] + test_files
+subprocess.run(cmd)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)